src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -43,6 +43,19 @@ public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
                 {
                     Proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
                 }
+                catch (DownloadClientException ex)
+                {
+                    if (ex.InnerException is HttpException && (ex.InnerException as HttpException).Response.StatusCode == HttpStatusCode.Conflict)
+                    {
+                        // category doesn't exist; add and try again
+                        Proxy.AddTorrentLabel(Settings.TvImportedCategory, Settings);
+                        Proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
+                    }
+                    else
+                    {
+                        throw;
+                    }
+                }
```

Comment:
This outputs `[Warn] HttpClient: HTTP Error - Res: [POST] http://localhost:8080/api/v2/torrents/setCategory: 409.Conflict` since we are allowing the conflict to happen. Not sure if we should just ignore it since we are expecting it or try a different route.

Reply:
I think this stems from https://github.com/Sonarr/Sonarr/blob/52aa84e9f959108ef2c2deefa73a92a7b0ae1531/src/NzbDrone.Common/Http/HttpClient.cs#L86-L101

Really to fix this it seems we could suppress errors for calls that we know the api will return errors, but then we don't get the exception thrown. Or we could just let the logging happen further up the stack and be dealt with on a case-by-case.